src/cron/cleanGiftedPlus.ts

```diff
@@ -0,0 +1,25 @@
+import { Cron } from './cron';
+import { User } from '../entity';
+
+const cron: Cron = {
+  name: 'clean-gifted-plus',
+  handler: async (con, logger) => {
+    logger.info('cleaning gifted plus membership...');
+    const timeThreshold = new Date();
+
+    const { affected } = await con
+      .getRepository(User)
+      .createQueryBuilder('user')
+      .update()
+      .set({ subscriptionFlags: {} })
+      .where('"user"."subscriptionFlags" ->> :expire  < :time', {
+        expire: 'giftExpirationDate',
```

Comment:
```suggestion
      .where('"user"."subscriptionFlags"->>'giftExpirationDate' < :time', {
```

Reply:
this part can be static right?

Reply:
We decided not to check if user didn't buy plus already?
Secondary thing comes to mind, a gifted/plus user can't be re-gifted before ends from FE?

Reply:
You cannot gift to a already-plus user, so no re-gift is possible :(

Reply:
> We decided not to check if user didn't buy plus already?
If user buy plus `giftExpirationDate` is cleaned so is not possible to delete "standard" subscription 👀